src/reflect/value.go

```diff
@@ -861,6 +861,11 @@ func (v Value) FieldByIndex(index []int) Value {
 	panic("unimplemented: (reflect.Value).FieldByIndex()")
 }
 
+// FieldByIndexErr returns the nested field corresponding to index.
+func (v Value) FieldByIndexErr(index []int) (Value, error) {
+	return Value{}, &ValueError{Method: "FieldByIndexErr"}
```

Comment:
This is probably the wrong error to return, but importing `errors` causing an import cycle. So it's good enough for the stub to compile, but maybe you'd want a more fleshed out error.

Reply:
Agreed that it is enough for now.